### ISSUE TITLE
Highlight search result

### ISSUE THREAD

**Issue by brerdem** [PDFTron/pdftron-react-native#315]
As the title says I would like to highlight matching text when I use `findText`. I'm also using `onTextSearchResult` and getting quad coordinates of the match but couldn't figure it out how to highlight it. I'd appreciate your help.

**Comment by sgong-pdftron** [comment]
@brerdem yes we are planning to add this, we will update you here when it is available. Thanks.

**Comment by sgong-pdftron** [comment]
@brerdem you can track the progress here: https://github.com/PDFTron/pdftron-react-native/pull/318. Thanks

**Comment by brerdem** [comment]
ok thanks!
